containers/java/.devcontainer/base.Dockerfile

```diff
@@ -1,7 +1,10 @@
 # This base.Dockerfile uses separate build arguments instead of VARIANT
-ARG TARGET_JAVA_VERSION=11
 ARG BASE_IMAGE_VERSION_CODENAME=bullseye
-FROM openjdk:${TARGET_JAVA_VERSION}-jdk-${BASE_IMAGE_VERSION_CODENAME}
+FROM debian:${BASE_IMAGE_VERSION_CODENAME}
```

Comment:
We can actually remove the "common" script from here by using `mcr.microsoft.com/vscode/devcontainers/base:${BASE_IMAGE_VERSION_CODENAME}`. These are the `debian` and `ubuntu` images with the common script already run and the user as `vscode`.